src/main/ml-modules/root/lib/facetsLib.mjs

```diff
@@ -16,72 +16,93 @@ import { BadRequestError } from './mlErrorsLib.mjs';
 import { FACETS_VIA_SEARCH_CONFIG } from '../config/facetsViaSearchConfig.mjs';
 import { facetToScopeAndTermName } from '../utils/searchTermUtils.mjs';
 
-// A safeguard from a current or future viaSearch facet request from pulling too many documents from disk.
-// Not a concern for the initial two facets but what about the next one?  As opposed to facets that pull
-// their values from a field, enabling a request to pull an unlimited number of documents from disk would
-// be bad.
-const MAXIMUM_FACET_VALUE_VIA_SEARCH_DOCUMENTS = 100;

```

Comment:
@clarkepeterf, can you find a place within this lib to include history for TTFU.  Perhaps at the top given the changes are sprinkled throughout.  Something similar to:

Two changes were made to restore system stability during performance tests: a) requests are now paginated and b) reduced the ratio between facet value and response body size.  For details, see #160 (non-semantic facet pagination), #161 (semantic facet pagination and facet response body change), and #162 (performance test).

Please also include rationale for the two page length limits.

Reply:
@brent-hartwig I've added a comment explaining rationale. Mind reviewing?